test/com/facebook/buck/apple/AppleCxxPlatformsTest.java

```diff
@@ -130,6 +130,7 @@ public void iphoneOSSdkPathsBuiltFromDirectory() throws Exception {
         .add(Paths.get("Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib"))
         .add(Paths.get("Toolchains/XcodeDefault.xctoolchain/usr/bin/strip"))
         .add(Paths.get("Toolchains/XcodeDefault.xctoolchain/usr/bin/nm"))
+        .add(Paths.get("Toolchains/XcodeDefault.xctoolchain/usr/bin/install_name_tool"))
```

Comment:
Please also add `install_name_tool` to `FakeAppleRuleDescriptions` so the tests can work.